packages/api-observable/src/Queries.ts

```diff
@@ -0,0 +1,207 @@
+    return this
+      .rawStorage(storage.democracy.votersFor, index)
+      .pipe(
+        // @ts-ignore After upgrade to 6.3.2
+        map((voters: Array<string> = []) =>
+          voters
+        )
+      );
+  }
+
+  democracyVotingPeriod = (): Observable<BlockNumber | undefined> => {
+    return this.rawStorage(storage.democracy.votingPeriod);
+  }
+
+  fees = (): Observable<RxFees> => {
+    return this
+      .rawStorageMulti(
+        [storage.balances.transactionBaseFee],
+        [storage.balances.transactionByteFee],
+        [storage.balances.creationFee],
+        [storage.balances.existentialDeposit ],
+        [storage.balances.transferFee ]
+      )
+      .pipe(
+        // @ts-ignore After upgrade to 6.3.2
+        map(([baseFee = new Balance(0), byteFee = new Balance(0), creationFee = new Balance(0), existentialDeposit = new Balance(0), transferFee = new Balance(0)]: [Balance | undefined, Balance | undefined, Balance | undefined, Balance | undefined, Balance | undefined]) => ({
+          baseFee,
+          byteFee,
+          creationFee,
+          existentialDeposit,
+          transferFee
+        }))
+      );
+  }
+
+  eraLastLengthChange = (): Observable<BlockNumber | undefined> => {
+    return this.rawStorage(storage.staking.lastEraLengthChange);
+  }
+
+  sessionReward = (): Observable<Perbill | undefined> => {
+    return this.rawStorage(storage.staking.sessionReward);
+  }
+
+  sessionRewardCurrent = (): Observable<Balance | undefined> => {
+    return this.rawStorage(storage.staking.currentSessionReward);
+  }
+
+  sessionCurrentIndex = (): Observable<BlockNumber | undefined> => {
+    return this.rawStorage(storage.session.currentIndex);
+  }
+
+  sessionCurrentStart = (): Observable<Moment | undefined> => {
+    return this.rawStorage(storage.session.currentStart);
+  }
+
+  sessionLastLengthChange = (): Observable<BlockNumber | undefined> => {
+    return this.rawStorage(storage.session.lastLengthChange);
+  }
+
+  sessionLength = (): Observable<BlockNumber | undefined> => {
+    return this.rawStorage(storage.session.sessionLength);
+  }
+
+  sessionsPerEra = (): Observable<BlockNumber | undefined> => {
+    return this.rawStorage(storage.staking.sessionsPerEra);
+  }
+
+  sessionValidators = (): Observable<Array<AccountId | string>> => {
+    return this
+      .rawStorage(storage.session.validators)
+      .pipe(
+        // @ts-ignore After upgrade to 6.3.2
+        map((validators: Array<AccountId> = []) =>
+          validators
+        )
+      );
+  }
+
+  stakingIntentions = (): Observable<Array<AccountId | string>> => {
+    return this
+      .rawStorage(storage.staking.intentions)
+      .pipe(
+        // @ts-ignore After upgrade to 6.3.2
+        map((intentions: Array<AccountId> = []) =>
+          intentions
+        )
+      );
+  }
+
+  stakingFreeBalanceOf = (address: AccountId | string): Observable<Balance | undefined> => {
```

Comment:
These need to be renamed - kept it as-is for the initial PR, i.e. just takin what is there and moving it to the correct storage area. (Actually started off renaming all, but in reality would first need to get the UI working and then start renaming)